compiler/aarch64/codegen/OMRMachine.cpp

```diff
@@ -314,12 +308,23 @@ TR::RealRegister *OMR::ARM64::Machine::freeBestRegister(TR::Instruction *current
       best = toRealRegister(candidates[0]->getAssignedRegister());
       }
 
-   TR::Register *registerToSpill = candidates[0];
-   TR_Debug *debugObj = cg->getDebug();
+   self()->spillRegister(currentInstruction, candidates[0]);
+   return best;
+   }
+
+void OMR::ARM64::Machine::spillRegister(TR::Instruction *currentInstruction, TR::Register *virtReg)
+   {
+   TR::Register *registerToSpill = virtReg;
+   TR::CodeGenerator *cg = self()->cg();
+   TR::Compilation *comp = cg->comp();
+   TR_Debug *debugObj = self()->cg()->getDebug();
+   TR_RegisterKinds rk = virtReg->getKind();
+   TR::Node * currentNode = currentInstruction->getNode();
```

Comment:
Please remove the space between `*` and `currentNode`.

Reply:
Removed the space.